packages/@react-aria/table/intl/en-US.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sortable": "sortable column",
+  "ascending": "ascending",
+  "descending": "descending",
+  "ascendingSort": "sorted by column {columnIndex} in ascending order",
+  "descendingSort": "sorted by column {columnIndex} in descending order"
```

Comment:
Rather than `columnIndex`, we should use the column name. For example, "sorted by Last Name in ascending order".

Reply:
Makes sense, I think I can grab the `textValue` prop provided to the column for this. What do you think about falling back to column index in case a `textValue` isn't provided?